types/setting.go

```diff
@@ -411,6 +414,19 @@ var (
 		},
 	}
 
+	SettingDefinitionBackupExecutionTimeout = SettingDefinition{
+		DisplayName: "Backup Execution Timeout",
+		Description: "In minutes. This setting determines the timeout for the backup engine execution. The default value is 1.",
```

Comment:
Waiting for the review of the description.

Reply:
Modified to `Number of minutes that Longhorn allows for backup execution. The default value is "1"`